### DISCUSSION TITLE
What is the sauce that traditionally accompanies ham and eggs on a muffin in eggs Benedict?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#403] [Galaxy Trivia] [answered]
**Hint:** 🇳🇱

**Comment by peterbaccaro** [comment]
**Hollandaise sauce** which is also called **Dutch sauce** which is a mixture of egg yolk, melted butter, and lemon juice.

**Comment by thuongtruong1009** [comment, marked as answer]
The sauce that traditionally accompanies ham and eggs on a muffin in Eggs Benedict is hollandaise sauce. Hollandaise sauce is a rich and creamy emulsion made with egg yolks, butter, lemon juice, and a pinch of cayenne pepper or hot sauce. The sauce is usually served warm and poured over the top of the ham and poached eggs that are placed on a toasted English muffin. The combination of the savory ham, runny egg yolks, and tangy hollandaise sauce creates a delicious and classic breakfast dish that is enjoyed all over the world.
